clients/drcachesim/common/options.cpp

```diff
@@ -97,7 +97,9 @@ droption_t<std::string> op_indir(
 droption_t<std::string> op_infile(
     DROPTION_SCOPE_ALL, "infile", "", "Offline legacy file for input to the simulator",
     "Directs the simulator to use a single all-threads-interleaved-into-one trace file. "
-    "This is a legacy file format that is no longer produced.");
+    "This is a legacy file format that is no longer produced as it does not support "
```

Comment:
Even though it may be a part of the legacy support, the `-infile` option is still useful even with the current traces, e.g., for analyzing the syscall template file which is a single file.

Reply:
Good point: rewrote to say:

```
    DROPTION_SCOPE_ALL, "infile", "", "Single offline trace input file",
    "Directs the framework to use a single trace file.  This could be a legacy "
    "all-software-threads-interleaved-into-one trace file, a core-sharded single "
    "hardware thread file mixing multiple software threads, or a single software "
    "thread selected from a directory (though in that case it is better to use "
    "-only_thread, -only_threads, or -only_shards).  This method of input does "
    "not support any features that require auxiliary metadata files. "
    "Passing '-' will read from stdin as plain or gzip-compressed data.");
```